client/src/MenuAuth/MenuAuth.js

```diff
@@ -187,7 +196,9 @@ export class MenuAuthView {
       }
     );
 
-    this.confidentialButton.onclick = function () {
+    this.editorButton.onclick = function () {
+      if (this.disabled) return;
```

Comment:
a modified client could launch the editor, but this is for another time I guess

Reply:
Oh yes it is actually quite simple 😓